test/src/watcher/watcher.test.ts

```diff
@@ -70,13 +70,20 @@ describe("WatcherAppointmentStateReducer", () => {
     when(appMock.id).thenReturn("app1");
     when(appMock.startBlock).thenReturn(0);
     when(appMock.endBlock).thenReturn(1000);
-    const appointment = throwingInstance(appMock);
 
     const db = LevelUp(EncodingDown<string, any>(MemDown(), { valueEncoding: "json" }));
     const blockStore = new BlockItemStore<IBlockStub & Logs>(db);
 
     const blockCache = new BlockCache<IBlockStub & Logs>(100, blockStore);
-    blocks.forEach(b => blockCache.addBlock(b));
+
+    before(async () => {
+        await blockStore.start();
```

Comment:
Should really call blockStore.stop() somewhere

Reply:
Done in 4d9060a.